site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -236,7 +236,14 @@ The following tables lists the configurable parameters of the Agones chart and t
 
 | Parameter                                           | Description                                                                                     | Default                |
 | --------------------------------------------------- | ----------------------------------------------------------------------------------------------- | ---------------------- |
-|                                                     |                                                                                                 |                        |             
+| `agones.controller.annotations`                     | Controller [annotations][annotations] settings for pod assignment                               | `{}`                   |
+| `agones.allocator.annotations`                      | Allocator [annotations][annotations] settings for pod assignment                                | `{}`                   |
+| `agones.ping.annotations`                           | Ping [annotations][annotations] settings for pod assignment                                     | `{}`                   |
+| `agones.ping.http.annotations`                      | [Annotations][annotations] for ping http service                                                | `{}`                   |
```

Comment:
Sorry to make you juggle this around, but `agones.ping.udp.annotations` and `agones.ping.http.annotations` already exist in the current release, but aren't documented - so they can go in the section up above.

There new, remaining values should stay down here.

Random thought - do we think there is any confusion given that `agones.ping.udp.annotations` is a annotation on a service, but `agones.controller.annotations` is on Pods?

Reply:
> Random thought - do we think there is any confusion given that agones.ping.udp.annotations is a annotation on a service, but agones.controller.annotations is on Pods?

Hmmm, it's somewhat confusing.
Should I revise it to `agones.controller.podAnnotations` or `agones.ping.udp.serviceAnnotations` or something?

Reply:
> Sorry to make you juggle this around, but agones.ping.udp.annotations and agones.ping.http.annotations already exist in the current release, but aren't documented - so they can go in the section up above.
>
> There new, remaining values should stay down here.

I understand.
I'll fix the name I asked for in my comment above when I decide what to do with it.

Reply:
@roberthbailey @pooneh-m would love a second opinion?

Worth noting, `agones.ping.udp.annotations` and `agones.ping.http.annotations` where in previous releases, but weren't documented.

WDYT?

Reply:
If you look at the structure in https://github.com/googleforgames/agones/blob/master/install/helm/agones/values.yaml I don't actually think it's all that confusing. The annotations for the service / pod are clearly nested under the values for the service / pod respectively. 

I think the descriptions in this table should make it clear what type of k8s resource the annotations are being applied to. e.g.

| `agones.controller.annotations` | [Annotations][annotations] added to the Agones controller pods |
| `agones.ping.http.annotations ` | [Annotations][annotations] added to the Agones ping http service |

Reply:
SGTM! Thanks for the second/third pair of eyes - that works for me :+1:

Reply:
Fixed :+1: